frontend/controller/actions/identity.js

```diff
@@ -155,6 +155,9 @@ export default (sbp('sbp/selectors/register', {
             router.push({ path: '/dashboard' }).catch(console.warn)
           }
         }
+      } else {
+        // when the user has already switched into the group, record the log-in time for the group.
```

Comment:
This doesn't quite say what you explained in your [previous comment](https://github.com/okTurtles/group-income/pull/1453#discussion_r1027051614).

If I were reading this comment for the first time, I would still be completely confused.

Something like this would be clearer:

```js
// We call updateLastLoggedIn in this else clause, instead of outside of it because
// in the `if` above, updateLastLoggedIn will get called by 'gi.actions/group/switch'
```

Reply:
it sounds better. yeah.